crates/snforge-scarb-plugin/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+use attributes::{
+    available_gas::AvailableGasCollector, fork::ForkCollector, fuzzer::FuzzerCollector,
+    ignore::IgnoreCollector, should_panic::ShouldPanicCollector,
+};
+use cairo_lang_macro::{attribute_macro, executable_attribute, ProcMacroResult, TokenStream};
+use config_statement::extend_with_config_cheatcodes;
+
+mod args;
+mod asserts;
+mod attributes;
+mod config_statement;
+mod parse;
+
+executable_attribute!("test");
+
+#[attribute_macro]
+fn ignore(args: TokenStream, item: TokenStream) -> ProcMacroResult {
+    extend_with_config_cheatcodes::<IgnoreCollector>(args, item)
```

Comment:
Do you really need all these generics? As far as I see, they are quite prevalent here, and I'm afraid of the code bloat that monomorphization could generate here.

I am almost sure it should be pretty straightforward to use `dyn` traits here instead of generics. Wdyt?

Reply:
Monomorphization doesn't affect it much (tested). Choosing between generics or trait objects is mostly a readability decision here. If you think that objects would be clearer, we can change it.

Reply:
If you have tested this then it's fine